.github/workflows/vast.yaml

```diff
@@ -513,32 +593,36 @@ jobs:
         uses: cachix/install-nix-action@v19
         with:
           nix_path: nixpkgs=channel:nixos-unstable
+      - name: Configure ssh-agent
+        uses: webfactory/ssh-agent@v0.7.0
+        with:
+          ssh-private-key: ${{ secrets.VAST_PLUGINS_DEPLOY_KEY }}
       - name: Setup Cachix
         uses: cachix/cachix-action@v12
         with:
           name: vast
           signingKey: "${{ secrets.CACHIX_VAST_SIGNING_KEY }}"
       - name: Build Static Packages
         id: build_static_packages
-        env:
-          STATIC_BINARY_TARGET: vast
-          VAST_BUILD_VERSION: ${{ needs.configure.outputs.build-version }}
-          VAST_BUILD_VERSION_SHORT: ${{ needs.configure.outputs.build-version-short }}
-        run: |
-          STORE_PATH=$(nix develop .#static-shell -c ./nix/static-binary.sh  ${{ github.event.inputs.arguments }} \
-            -DCPACK_PACKAGE_FILE_NAME:STRING="vast-${VAST_BUILD_VERSION}")
-          echo "store_path=${STORE_PATH}" >> $GITHUB_OUTPUT
-      - name: Get Artifact Names
-        id: get_artifact_names
         run: |
-          TAR_GZ=$(ls "${{ steps.build_static_packages.outputs.store_path }}" | grep "vast.*.tar.gz")
-          DEB=$(ls "${{ steps.build_static_packages.outputs.store_path }}" | grep "vast.*.deb")
+          STORE_PATH=$(nix --print-build-logs build .#${{ matrix.vast-nix.package }}.package --print-out-paths)
+          TAR_GZ=$(find "$STORE_PATH" -type f -name '*.tar.gz')
+          echo "TARBALL is at: $TAR_GZ"
           echo "tar_gz=${TAR_GZ}" >> $GITHUB_OUTPUT
+          DEB="$(find "$STORE_PATH" -type f -name '*.deb')"
+          echo "DEB is at: $DEB"
           echo "deb=${DEB}" >> $GITHUB_OUTPUT
+          # The debian package filename contains a colon (:), but GitHub does
+          # not allow this character in attached assets, so we create an extra
+          # copy.
+          DEB_GITHUB="$PWD/$(basename "$DEB" | tr ':' '_')"
+          echo "DEB_GITHUB is at: $DEB_GITHUB"
```

Comment:
The package name must be set so it's either beginning with vast or vast-ce; currently it's always the former, so there's a race in CI for the upload.